sql/moz-fx-data-shared-prod/fivetran_costs_derived/monthly_connector_costs_v1/query.sql

```diff
@@ -43,13 +43,13 @@ monthly_connector_ratios AS (
     connector_mar.free_active_rows,
     connector_mar.total_active_rows,
     ROUND(
-      connector_mar.paid_active_rows / NULLIF(destinations_total_mar.paid_active_rows, 0) * 100,
+      connector_mar.paid_active_rows / NULLIF(destinations_total_mar.paid_active_rows, 0),
       2
-    ) AS percentage_of_destination_paid_active_rows,
+    ) AS ratio_destination_paid_active_rows,
     ROUND(
-      connector_mar.paid_active_rows / NULLIF(total_mar.paid_active_rows, 0) * 100,
+      connector_mar.paid_active_rows / NULLIF(total_mar.paid_active_rows, 0),
       2
-    ) AS percentage_of_total_active_rows
+    ) AS ratio_total_active_rows
```

Comment:
- Rounding these ratios to 2 decimal places is like rounding to whole percentages, which will be much less accurate.  I'd recommend not rounding them at all.
  - If you take this suggestion you will want to round the later calculated percentages.
- Suggest renaming `ratio_total_active_rows` to `ratio_total_paid_active_rows`.